spec/models/project_step_spec.rb

```diff
@@ -36,4 +36,20 @@
     expect(step[:original_date]).not_to be_nil
   end
 
+  context 'groups' do
```

Comment:
Not sure what to do for a name. I typically only have one expectation for each `it` block, and use contexts instead of multiple expects.